### PR TITLE
Updating deployment - custom scripts area

### PR DESCRIPTION
**Pull request by shaun-od**
# Background

Updating screenshots and adding icons to the customs scripts section of the deployments area as part of the docs refresh.

# Results

- Updated screenshots, last modified dates, page icons
- Added older version sections

### CODE REVIEW

File: dictionary-octopus.txt [hunk 1]
```diff
@@ -12,6 +12,7 @@ approot
 appsettings
 Appstep
 Appsteps
+argparse
```
Comment (shaun-od) [inline dictionary-octopus.txt:15]: These were needed to be added as they are in the code samples and can't be changed

File: src/pages/docs/deployments/custom-scripts/logging-messages-in-scripts.md [hunk 2]
```diff
@@ -98,7 +99,7 @@ Progress messages will display and update a progress bar on your deployment task
 
 ```ps PowerShell
 Update-Progress 10
-Update-Progress 50 "Woah, we're halfway there!"
+Update-Progress 50 "We're halfway there!"
```
Comment (shaun-od) [inline src/pages/docs/deployments/custom-scripts/logging-messages-in-scripts.md:102]: All the changes in this file were needed to satisfy spelling issues.

File: src/pages/docs/deployments/custom-scripts/output-variables.md [hunk 3]
```diff
@@ -1,22 +1,15 @@
...
+modDate: 2024-08-27
 title: Output variables
 description: Your scripts can emit variables that are available in subsequent deployment steps.
+icon: fa-solid fa-file-export
 navOrder: 70
 ---
 
 Your scripts can emit variables that are available in subsequent deployment steps. This means you can factor your deployment into smaller, more well-defined steps that leverage the result of prior steps. It is an extremely powerful feature and you should refer to the documentation on [output variables](/docs/projects/variables/output-variables) for more information.
 
-This example is from the sample project in the [Channels Walk-through](https://octopus.com/blog/channels-walkthrough#prerequisites).
```
Comment (shaun-od) [inline src/pages/docs/deployments/custom-scripts/output-variables.md:12]: I felt this screenshot was not adding any value to this page.

File: src/pages/docs/deployments/custom-scripts/passing-parameters-to-scripts.md [hunk 4]
```diff
@@ -43,7 +44,7 @@ You can define your parameters in the **Script Parameters** field using the form
 Don't forget to correctly delimit your parameters correctly for the scripting engine. In the example above we have surrounded the parameter value in double-quotes to handle cases where the Environment Name has spaces: `"#{Octopus.Environment.Name}"`
 :::
 
-## Passing parameters to PowerShell scripts {#Customscripts-PassingparameterstoPowerShellscripts}
+## Passing parameters to PowerShell scripts {#Customscripts-PassingParametersToPowerShellScripts}
```
Comment (shaun-od) [inline src/pages/docs/deployments/custom-scripts/passing-parameters-to-scripts.md:47]: These changes were needed to satisfy the spelling check

File: src/pages/docs/deployments/custom-scripts/script-modules.md [hunk 5]
```diff
@@ -1,17 +1,18 @@
...
+icon: fa-regular fa-file-code
 navOrder: 120
 ---
 
 Script modules allow users to create collections of language specific functions that can be used in deployment processes across multiple projects.
 
 ## Creating a script module {#create-script-module}
 
-1.  Navigate to **Library ➜ Script Modules ➜ Add Script Module**.
+1.  Navigate to **Deploy ➜ Script Modules ➜ Add Script Module**.
```
Comment (shaun-od) [inline src/pages/docs/deployments/custom-scripts/script-modules.md:15]: The navigation has changed in the new UI.

File: src/pages/docs/deployments/custom-scripts/using-variables-in-scripts.md [hunk 6]
```diff
@@ -84,7 +85,7 @@ print(connectionString)
 To see the F# API available to your F# scripts, take a look at our [F# signature file](https://github.com/OctopusDeploy/Calamari/tree/master/source/Calamari.Common/Features/Scripting/FSharp/Bootstrap.fsi).
 :::
 
-## Variables in PowerShell scripts {#Customscripts-VariablesinPowerShellscripts}
+## Variables in PowerShell scripts {#Customscripts-VariablesInPowerShellScripts}
```
Comment (shaun-od) [inline src/pages/docs/deployments/custom-scripts/using-variables-in-scripts.md:88]: Again to fix issues with the spelling check.

**Review by steve-fenton-octopus — APPROVED** [review]
LGTM - I adjusted some bookmark links and checked for references just to bring them into the right style.

**Pull request merged**